usr/share/rear/skel/NBKDC/etc/scripts/system-setup.d/65-start-nbkdc-client.sh

```diff
@@ -0,0 +1,46 @@
+# start included NBK DC client so file restore can happen:
+
+echo "Sourcing $SHARE_DIR/lib/nbkdc-functions.sh"
+source $SHARE_DIR/lib/nbkdc-functions.sh
+
+# Check if working directories of NBK DC datamover exist
+if [ ! -d $NBKDC_HIBLST_DIR ]; then
+	mkdir $NBKDC_HIBLST_DIR
```

Comment:
Please do that in the mkrescue stage where you can manipulate the rootfs of the rescue system.